lib/chf/data_fixes/work_dates.rb

```diff
@@ -0,0 +1,26 @@
+module CHF
+  module DataFixes
+    class WorkDates
+      attr_reader :work
+
+      def initialize(work)
+        @work = work
+      end
+
+      # possibly makes changes to work (without saving), returns true
+      # if changes were made, so caller can do expensive save.
+      def change
+        updated = false
```

Comment:
interesting idea, but especially if we don't know if it's reliable, i'm happy with how i've done it. this is reporting if THIS operation made a change specifically, too. Although if we could rely on `changed?` it wouldn't have to. But I think this is good enough, the hackiness is localized.